README.md

```diff
@@ -57,17 +50,26 @@ To start Express, run the following from the root directory:
 
 This will run the express server on `http://localhost:3000`
 
-## Run Client-Side Tests
+### 6. Local HTTP-Proxy to Facet Service
+
 
-Run the following from the `spa` directory:
+The URL for the back end Azul Facet Service is configurable from the environment property ```BW_DATA_URL```. This can be set to the url of the server running the back end in the Gruntfile to run locally or on the boardwalk node.js server.
 
-	npm test
-	
-This will run once through test-suite using PhantomJS. To run through PhantomJS with a file watcher, you can use:
+If this value is not set, the Boardwalk Angular client will assume the Facet Server is at the same url that served the front end, and make a relative request to /api/v1 to retrieve the facets and other configuration. 
 
-    npm test:headless
+To support this, on localhost, run the proxy.json file with `node proxy.json` This will start a proxy server at port 3001. ```proxy.conf.json``` in ```/spa``` is configured to forward requests to /api to the local node.js server at port 3000 and to send /api/v1 to requests to the proxy listening at port 3001.
 
-To run the tests through the browser with a file watcher:
+The HTTP-Proxy on port 3001 is configured to send all requests to https://ucsc-cgp.org but this can be chaned as required.
 
-    npm test:browser
 
+``````
+//
+// Create a HTTP Proxy server with a HTTPS target
+//
+httpProxy.createProxyServer({
+    target: 'https://ucsc-cgp.org',
```

Comment:
Given that this is the feature/commons branch, I think you should be default point to our Commons deploy instance, `https://commons.ucsc-cgp-dev.org`. This will eventually matter, as only the Commons instance will support the new endpoints, e.g., `/me`, that you will be relying on.